layouts/partials/head.html

```diff
@@ -24,7 +24,7 @@
 {{- template "_internal/opengraph.html" . -}}
 {{- template "_internal/schema.html" . -}}
 {{- template "_internal/twitter_cards.html" . -}}
-{{- if hugo.IsProduction -}}
+{{- if in hugo.Environment "prod" -}}
```

Comment:
Can you explain why `hugo.IsProduction` isn't the right test to use? Maybe add a comment here?

Also, perhaps we could logical-OR the two tests and serve the analytics if either is true?

Reply:
Thanks for the feedback @sftim. Actually, your comments make me think that we have a broader issue here. Ok, let's hold this while I put together an alternative proposal.

/hold

Reply:
I've created the following for further discussion:

- #36333
- #36334